src/content/changes/highlights/2023-06-15.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-06-15T14:00:00
+title: Highlights for the week ending June 15 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+- [linkerd2-cni](https://github.com/giantswarm/linkerd2-cni-app) version [v1.0.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#100---2023-06-13) includes `Linkerd v2.13.4` being our first stable app release.
+
+- [linkerd-control-plane](https://github.com/giantswarm/linkerd-control-plane-app) version [v1.0.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#100---2023-06-13) includes `Linkerd v2.13.4` being our first stable app release.
+
+- [linkerd-viz](https://github.com/giantswarm/linkerd-viz-app) version [v1.0.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#100---2023-06-13) includes `Linkerd v2.13.4` being our first stable app release.
+
+
```

Comment:
_Info_: For more information about the `Linkerd v2.13.4` changes take a look to the [official release notes](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.4).